ui/app/components/mount-backend-form.js

```diff
@@ -56,11 +56,14 @@ export default Component.extend({
   }),
 
   engines: computed('version.features[]', function() {
+    let mountEngines = [...ENGINES];
     if (this.version.hasFeature('KMIP')) {
-      return ENGINES.concat([KMIP]);
-    } else {
-      return ENGINES;
+      mountEngines.push(KMIP);
     }
+    if (this.get('version.isEnterprise')) {
+      mountEngines.push(TRANSFORM);
```

Comment:
As I understand it, KMIP and Transform are enabled under the same module. So, we could treat KMIP the same way (shown for enterprise, but disabled unless the feature exists). However, that wasn't outlined in the designs to I kept it this way for now.